prov/efa/src/rxr/rxr_ep.c

```diff
@@ -743,9 +744,11 @@ static int rxr_ep_close(struct fid *fid)
 		retv = ret;
 	}
 
-	ret = fi_close(&rxr_ep->rdm_cq->fid);
```

Comment:
I think the proper place to release the `scq` and `rcq` allocated in `rxr_ep_setup_device_cq` is here. Maybe introduce another function like `rxr_ep_release_device_cq` to do that.

Reply:
fixed